src/pages/Login.tsx

```diff
@@ -0,0 +1,11 @@
+import React, { ReactElement } from "react";
+import { Link } from "react-router-dom";
+
+export default (): ReactElement => {
```

Comment:
The following syntax is preferred.
Here the return type is Implicit, it's too verbose, typescript is checking the return type on its own. Use the return value only when necessary.
```
`const Login = () => {
	  return (
             <div>
               <h1>Login Page</h1>
               <Link to='/register'>Register Page</Link>
             </div>
  );
};

export default Login;`
```

Reply:
Done

Reply:
`Here the return type is Implicit, it's too verbose, typescript is checking the return type on its own. Use the return value only when necessary.`